<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крестики-Нолики</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../media/andrey/3Q HDD External/favicon.ico</normaloff>../../../../../../../media/andrey/3Q HDD External/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Первый ход</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="rPC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ПК</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rUser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>61</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Юзверь</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rUvsU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1+1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>ИГРОВОЕ ПОЛЕ, МАТЬ ВАШУ</string>
    </property>
    <widget class="QPushButton" name="btn00">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn01">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn02">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn11">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn21">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn22">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="log">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Логирование</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblWind">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>190</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>rPC</tabstop>
  <tabstop>rUser</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btn00</tabstop>
  <tabstop>btn01</tabstop>
  <tabstop>btn02</tabstop>
  <tabstop>btn10</tabstop>
  <tabstop>btn11</tabstop>
  <tabstop>btn12</tabstop>
  <tabstop>btn20</tabstop>
  <tabstop>btn21</tabstop>
  <tabstop>btn22</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
